src/Tizen.Multimedia.Vision/MediaVision/RoiTracker.cs

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading.Tasks;
+using InteropRoi = Interop.MediaVision.RoiTracker;
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Provides the ability to track ROI(Region Of Interest) area on image source.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public static class RoiTracker
+    {
+        /// <summary>
+        /// Tracks ROI(Region Of Interest) on the source image.<br/>
+        /// </summary>
+        /// <feature>http://tizen.org/feature/vision.roi_tracking</feature>
+        /// <param name="source">The source of the media user wants to track.</param>
+        /// <param name="config">The engine's configuration that will be used for tracking.</param>
+        /// <param name="roi">The ROI to track.</param>
```

Comment:
what is the relation between roi and config.Roi?

Reply:
It's mistake. I'll remove the `roi` parameter.